cpp/benchmarks/io/parquet/parquet_reader_input.cpp

```diff
@@ -56,34 +56,30 @@ void parquet_read_common(cudf::io::parquet_writer_options const& write_opts,
   state.add_buffer_size(source_sink.size(), "encoded_file_size", "encoded_file_size");
 }
 
-template <data_type DataType, cudf::io::io_type IOType>
-void BM_parquet_read_data(
-  nvbench::state& state,
-  nvbench::type_list<nvbench::enum_type<DataType>, nvbench::enum_type<IOType>>)
+template <data_type DataType>
+void BM_parquet_read_data(nvbench::state& state, nvbench::type_list<nvbench::enum_type<DataType>>)
 {
-  auto const d_type                 = get_type_or_group(static_cast<int32_t>(DataType));
-  cudf::size_type const cardinality = state.get_int64("cardinality");
-  cudf::size_type const run_length  = state.get_int64("run_length");
-  auto const compression            = cudf::io::compression_type::SNAPPY;
+  auto const d_type                   = get_type_or_group(static_cast<int32_t>(DataType));
+  cudf::size_type const cardinality   = state.get_int64("cardinality");
+  cudf::size_type const run_length    = state.get_int64("run_length");
```

Comment:
If you don't see any warning now then that should be a compiler bug---the compiler may change its behavior for this case anytime in the future. So please address it now instead of waiting until the warning shows up.

Reply:
Thanks for noticing @ttnghia!